query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala

```diff
@@ -125,6 +123,24 @@ class FilingComponentSpec extends AsyncWordSpec with MustMatchers with FilingCom
 
     }
 
+    "Stream IRS" in {
```

Comment:
That should be handled on the `UploadPaths`, which deletes the tables when a new file is uploaded.

See [here](https://github.com/cfpb/hmda-platform/blob/master/api/src/main/scala/hmda/api/http/institutions/UploadPaths.scala#L64)